src/index.test.js

```diff
@@ -1,4 +1,14 @@
 import * as ol_kit from './index' // eslint-disable-line camelcase
+import Map from 'Map'
+
+// utility used by many tests for wrapping a Map component
+export const mountOpts = props => ({
+  wrappingComponent: Map,
+  wrappingComponentProps: {
+    allowAsyncMount: false, // this forces wrappingComponent to render children immediately
+    ...props
+  }
+})
```

Comment:
should this be named `mapWrapper` or something more specific than `mountOpts`?

Reply:
We can drop these mount opts now that we've got async tests working right?